### PR TITLE
In Progress - Work Sites and Employees a11y, styling, and text updates

### PR DESCRIPTION
**Pull request by ltsheu**
#### What's this PR do?

Merge in-progress updates for Work Sites & Employees, to facilitate a11y testing and move some user stories forward.

- Style the sliding panel
- a11y updates to sliding panel
- a11y updates to tabs, hidden panels, and other components
- a11y updates to form fields
- a11y updates to help button/text
- Add new help buttons/text to Add Employee fields
- Update existing help text
- Style various components in the Add Employee tab (such as Help callout)
- Style the data tables and update data table icons

#### What are the relevant tickets?

#297, #304, #425

### CODE REVIEW

**Comment by ltsheu** [comment]
@mgwalker Could you please fast track PR #437 on Monday? :smile: It is a lot of changes, mainly in Work Sites and Employees. Please let me know if you have any questions!

File: DOL.WHD.Section14c.Web/src/modules/components/resultsTable/resultsTableController.js [hunk 1]
```diff
@@ -16,10 +16,11 @@ module.exports = function(ngModule) {
     'use strict';
 
     var vm = this;
+    var tableId = $scope.tableId;
     $scope.data = [];
 
     this.initDatatable = function () {
-      let exampleId = $('#example');
+      let exampleId = $('#' + tableId);
```
Comment (mgwalker) [inline DOL.WHD.Section14c.Web/src/modules/components/resultsTable/resultsTableController.js]: Elsewhere in the code, we are using [ES6-style template strings](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals).  For consistency's sake, this should be``$(`#${tableId}`)``

File: DOL.WHD.Section14c.Web/src/modules/components/resultsTable/resultsTableController.js [hunk 2]
```diff
@@ -87,13 +88,13 @@ module.exports = function(ngModule) {
 
     this.initDatatable();
 
-    $('#example tbody').on('click', 'td.edit-table-entry', function ($event) {
+    $('#' + tableId + ' tbody').on('click', 'td.edit-table-entry', function ($event) {
```
Comment (mgwalker) [inline DOL.WHD.Section14c.Web/src/modules/components/resultsTable/resultsTableController.js]: In this case:  ``$(`#${tableId} tbody`)``.  From here on, I'll just note strings that should be fixed.

File: DOL.WHD.Section14c.Web/src/modules/components/resultsTable/resultsTableController.js [hunk 3]
```diff
@@ -87,13 +88,13 @@ module.exports = function(ngModule) {
...
         var tr = $(this).closest('tr');
         var row = $scope.tableWidget.row( tr );
         $scope.$parent.vm.editEmployee(row[0][0], $event);
     } );
 
-    $('#example tbody').on('click', 'td.delete-table-entry', function ($event) {
+    $('#' + tableId + ' tbody').on('click', 'td.delete-table-entry', function ($event) {
```
Comment (mgwalker) [inline DOL.WHD.Section14c.Web/src/modules/components/resultsTable/resultsTableController.js]: String needs fixed

File: DOL.WHD.Section14c.Web/src/modules/components/sectionWageData/sectionWageDataController.js [hunk 4]
```diff
@@ -83,18 +82,23 @@ module.exports = function(ngModule) {
...
-    $('.dol-btn-helpful-links').on('click', function(event){     
-      $('.cd-panel').addClass('is-visible');
-      $('#helpfulLinksHeader').focus();
+    var panelTrigger    
+
+    $('.cd-panel-trigger').on('click', function(event){ 
+      panelTrigger = $(this);
+      var target = $(this).attr('aria-controls');    
+      $('#' + target).addClass('is-visible');
+      $('#' + target + ' .cd-panel-header h3').focus();
```
Comment (mgwalker) [inline DOL.WHD.Section14c.Web/src/modules/components/sectionWageData/sectionWageDataController.js]: Strings needs fixed

File: DOL.WHD.Section14c.Web/src/modules/components/sectionWorkSites/sectionWorkSitesController.js [hunk 5]
```diff
@@ -303,5 +297,63 @@ module.exports = function(ngModule) {
...
+    };
+    
+    // Sliding Panel
+    var panelTrigger     
+
+    $('.cd-panel-trigger').on('click', function(event){ 
+      panelTrigger = $(this);
+      var target = $(this).attr('aria-controls');    
+      $('#' + target).addClass('is-visible');
+      $('#' + target + ' .cd-panel-header h3').focus();
```
Comment (mgwalker) [inline DOL.WHD.Section14c.Web/src/modules/components/sectionWorkSites/sectionWorkSitesController.js]: Strings needs fixed

**Comment by mgwalker** [comment]
I'm going to leave the CodeClimate issues because I think ideally we'll refactor that duplicated code, but it's fine for now!

**Pull request merged**